### DISCUSSION TITLE
Bootstrap plots

### DISCUSSION THREAD

**Discussion by palday** [palday/MixedModelsMakie.jl#34] [Ideas]
`coefplot` will generate coefplots from bootstrap samples, but maybe there are a few other plots we could do. 

Maybe something like coefplot but with distributions/histograms/kde instead of / in addition to the intervals? What's called a "ridge plot" over in the Bayesian world.

Potentially something like a partial effects plot that plots the fitted line + a sample of lines based on the parameters from the bootstrap for each coefficient (I'm thinking of tiling like we do in `shrinkageplot!`). This would be similar to the posterior predictive distribution used in Bayesian diagnostics and would provide a visualization of the uncertainty around the original fit.

Something showing off the covariance of different parameters? 

cc @dmbates

**Comment by dmbates** [comment]
That is sort-of what I had in mind, although with my background I don't see much in the way of `coefplot`.  I think they are more prominent in your area.  I will spend some time tomorrow working on density plots and perhaps 2-dimensional density plots.

We should also consider plots from parameter trajectories and the objective when `thin` is set to a small value.

**Reply by palday** [reply]
> We should also consider plots from parameter trajectories and the objective when thin is set to a small value.

Most of that code is already in the `SMLP2021` notebook I created for the animations. I'll convert it to usable functions when I get a chance. :smile:  

I agree `coefplot` is pretty trivial, but a lot of researchers prefer seeing those things as plots instead of tables. :shrug:
